src/components/ContentResult.jsx

```diff
@@ -25,7 +25,10 @@ import { filterContactsBySearch, delayedSetThreshold } from '../helpers/search'
 const groupsSelectCustomStyles = {
   container: base => ({
     ...base,
-    width: '100%'
+    width: '50%',
+    '@media (max-width: 48rem)': {
```

Comment:
we avoid to use hardcoded breakpoint, you can use u-classes instead on `ControlDefault` I think, or an other component styled with makeStyle perhaps. Or maybe with style from app and so css classes